satellite/audit/worker.go

```diff
@@ -78,6 +102,45 @@ func (worker *Worker) process(ctx context.Context) (err error) {
 }
 
 func (worker *Worker) work(ctx context.Context, path storj.Path) error {
-	// TODO: handle work.
-	return nil
+	var errlist errs.Group
+	report, err := worker.verifier.Reverify(ctx, path)
```

Comment:
nit - add a comment: `First, attempt to reverify nodes for this segment that are in containment mode`